manifests/init.pp

```diff
@@ -114,10 +114,12 @@
   $prd_web_ui_logger_syslog_errors = true,
   $prd_web_ui_port                 = '3080',
   $prd_web_ui_timeout              = '300',
+  $prd_web_ui_base_url             = undef,
 
   ## JSON API Params
   $prd_json_api_access_log           = 'jsonapi_access.log',
   $prd_json_api_base_url             = 'http://localhost',
+  $prd_json_api_url                  = undef,
```

Comment:
I think that they're both used in the source code for values that will be used externally -- in pages or Location HTTP headers as appropriate. (Unless the proxy is doing some sort of translation I guess? I'd have to review things in that case.)